### ISSUE TITLE
Accessibility: Fix Joyce Brothers Issues

### ISSUE THREAD

**Issue by JesiBuell79** [cul-it/exhibits-library-cornell-edu#638]
**Describe the bug**
[A clear and concise description of the accessibility issue with the curated content.
](https://docs.google.com/spreadsheets/d/13T6FY0HVWu3fU2LXjGnv2VpvENSxh3AdVST4u_1H9Bs/edit#gid=0)

Exhibit: Joyce Brothers
Page: https://exhibits.library.cornell.edu/dr-joyce-brothers 
Content: information identifying the content that needs accessibility improvements

**To Reproduce**

**Expected behavior**

**Screenshots**

**Additional context**

**Comment by sarah-cul** [comment]
Found on All Pages:
- [ ] IN PAGE ANCHOR NOT FOUNDIn-page anchor target with [id=search_field] not found. This is referring to the invisible anchor “Skip to search” found at the top, the href is pointing to a hidden input element.Possibly this should point to the search box name="q" id="q". Note - There is no problem with the invisible anchor right below ‘Skip to Main Content’ it because it links to a visible element “main-container”.
- [ ] CONTRAST RATIO Text does not meet minimum AA contrast ratio (4.5:1)Links on menu (Home, Themes, About)     Link color: #4b5474     Suggested color: #464f6f
- [ ] CONTRAST RATIO Text does not meet minimum AA contrast ratio (4.5:1)Links on the footer (@2023, Privacy, Web Accessibility Assistance, Admin Login     Link color: #0068ac     Suggested color: #005094
- [ ] AUTOCOMPLETE attribute is empty.  From what I'm reading, WCAG 2.1 Level AA standards required autocomplete to be used. For screen readers to work properly, the autocomplete attribute values must be true and applied correctly. In the absence of the necessary autocomplete attribute values, screen readers will not read the identified autocomplete form fields. There are violations throughout the site where the autocomplete attribute does not have a value specified.

Found on EARLY LIFE:
- [ ] At the bottom, the link “Genetic Psychology Monographs.” has no href, id or tab index.
- [ ] 14 Ambiguous links: The collection of images, they all have a unique href, but the alt text is the same for all the thumbnails alt="Cornell University Orientation 100 term paper …”

Found on GAMES:
- [ ] [alt] attribute length exceeds 250 characters; consider condensing. Image “Dr. Brothers waves a check for $64,000 while being held on the shoulders of former boxi…”
- [ ] 12 Ambiguous links: The same issue as found in the EARLY LIFE page concerning thumbnails, the href are unique but the alt text is not. Maybe add a count to the alt text.

Found on SPORTS:
- [ ] 11 Ambiguous links. Same issue with the thumbnails and alt text.

Found on RADIO:
- [ ] 28 Ambiguous links, same issue with the thumbnails and alt text.

Found on TELEVISION:
- [ ] 20 Ambiguous links, same issue with the thumbnails and alt text.

Found on FASHION:
- [ ] 5 Ambiguous links, same issue with the thumbnails and alt text.

Found on PUBLICATIONS:
- [ ] 21 Ambiguous links, same issue with the thumbnails and alt text.

Found on ACTIVITIES:
- [ ]  8 Ambiguous links, same issue with the thumbnails and alt text. National Lampoon, J. Mansfield, Mother Goose

Found on LEGACY:
- [ ] 2 Ambiguous links, same issue with the thumbnails and alt text. Photograph of Brothers as the XVI Assembly of the Inter-American

Found on SEARCH RESULTS:
- [ ] CONTRAST RATIO: Text does not meet minimum AA contrast ratio (4.5:1).Blue ‘Start Over’ button with white text, adjust the blue slightly to pass     Background Color: #007bff     Suggested Background Color: #0070f4     class="catalog_startOverLink btn btn-primary”
- [ ] CONTRAST RATIO: Text does not meet minimum AA contrast ratio (4.5:1).Search string is shown in white box, same with the X button.  Adjust the gray font color of to make it pass.      Text Color:  #6c757d      Suggested Text Color:  #6b747c     class="filter-value" class="remove-icon"

**Comment by sarah-cul** [comment]
all content changes are done
